### ISSUE TITLE
Remove requirement that parent email be different than student email for all divisions

### ISSUE THREAD

**Issue by shaun-technovation** [Iridescent-CM/technovation-app#3771]
If a beginner student returns as a junior student, they will be unable to get agree to the data terms because as a junior student their email address can't match their parent/guardian's email address (which is allowed in the beginners division), this validation check fails and causes the "Uh oh, something went wrong" page.

**Comment by dboyer** [comment]
Note: we have three examples of this issue reported by a ChA from India. I confirmed on QA (account with the email deb+parent0811@). I was able to log into my account and was taken to the data terms page. When I selected the box agreeing to the terms and clicked the submit button, the data terms page just reloaded again.

<img width="1435" alt="Screen Shot 2023-01-06 at 4 37 56 PM" src="https://user-images.githubusercontent.com/516764/211104362-2d414b0d-a571-418e-ad18-86348c84bc32.png">

**Comment by dboyer** [comment]
Possible solutions:

1. Allow junior division user emails to also match their parent emails
2. Add logic allowing only students who move from beginner division to junior division in a year to have user emails that match parent emails (although this just kicks the can down the road a year)
3. Add a page that if the validation fails for this reason, we take the student to a page where they have to update either their user email or their parents email so they don't match

**Comment by dboyer** [comment]
Program team agrees that we should remove the requirement that the parent email be different than the user email. This can apply to all divisions.

**Comment by dboyer** [comment]
Validation is enforced in the junior/senior registration workflow and on the parent email page of the junior/senior dashboard.

**Comment by dboyer** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Technovation-605271851956b1000ec2f24d/issues/Iridescent-CM/technovation-app/3771?planning-poker) with Zenhub @shaun-technovation @viviancan

**Comment by dboyer** [comment]
Tested on QA:

- Registered a new junior account and used the same address for parent email and student email during registration. Did not receive any error message when trying to register. I received both the welcome email that goes to students and the parent consent email. After signing the forms, I also received both the student notification that my parents had signed the permission forms and the parent notification with copies of the signed consent forms. 

- Registered a new senior account and used the same address for parent email and student email during registration. Did not receive any error message when trying to register. I received both the welcome email that goes to students and the parent consent email. After signing the forms, I also received both the student notification that my parents had signed the permission forms and the parent notification with copies of the signed consent forms. 

- Registered a new beginner account and the email for the parent was automatically used for the account. Received all emails as expected and was able to create a team.

- Tried to register an account that used an existing student email for the parent email and received an error message that that was not possible. Tried to register with a student email address that was already in use for another student account and received an error message that I could not do so.

- Found an account from last season that was in the beginner division, logged in, and updated their age so they were now in the junior division. I was able to do that without any errors.
